Musakai.Blazor/Pages/Home.razor

```diff
@@ -32,6 +33,39 @@
     <PortfolioCard Src="" Title="Street Fotografie" />
     <PortfolioCard Src="" Title="Svatby" />
 </section>
+<section id="contact">
+    <div class="contact-info">
+        <h2>Contact Information</h2>
+        <p><strong>Email:</strong> example@example.com</p>
+        <p><strong>Phone:</strong> +123 456 7890</p>
+        <p><strong>Address:</strong> 123 Example Street, City, Country</p>
+        <p><strong>Bank Account:</strong> 1234 5678 9012 3456</p>
```

Comment:
Why?

Reply:
There should be only Email, Phone, Bank Account

Reply:
But why the bank account?

Reply:
Should not be displayed. Will be available in the invopice system.